src/logger/serial.rs

```diff
@@ -150,6 +154,19 @@ impl<T: io::Write> SerialLogger<T> {
         Ok(())
     }
 
+    fn write_duration(&self, buffer: &mut T, duration: Duration) -> io::Result<()> {
+        let seconds = duration.num_seconds();
```

Comment:
Milliseconds would be a better precision. None of my tests run in more than 1 second :)

Reply:
Fixed.